src/events/cd-event-list.vue

```diff
@@ -31,8 +31,11 @@
         </div>
       </div>
       <div v-if="canBook" class="cd-event-list__event-view-wrapper">
+        <div v-if="event.eventbriteId">
+          <a :href="event.eventbriteUrl | cdUrlFormatter" target="_blank" class="btn btn-lg btn-primary cd-event-list__event-view">{{ $t('See Details and Book') }}</a>
+        </div>
         <router-link :to="{name: 'EventDobVerification', params: {eventId: event.id}}"
```

Comment:
Really minor, but you could use `v-else` here instead of another `v-if`.

https://vuejs.org/v2/guide/conditional.html#v-else